sumpy/expansion/m2l.py

```diff
@@ -753,26 +894,84 @@ def translation_classes_dependent_data(self, tgt_expansion, src_expansion,
 
         first_column_circulant = list(first_column_toeplitz) + \
                 list(reversed(first_row_toeplitz))
-        return fft(first_column_circulant, sac)
+        return first_column_circulant
 
     def preprocess_multipole_exprs(self, tgt_expansion, src_expansion,
             src_coeff_exprs, sac, src_rscale):
 
-        from sumpy.tools import fft
         result = super().preprocess_multipole_exprs(tgt_expansion,
             src_expansion, src_coeff_exprs, sac, src_rscale)
 
         result = list(reversed(result))
         result += [0] * (len(result) - 1)
-        return fft(result, sac=sac)
+        return result
 
     def postprocess_local_exprs(self, tgt_expansion, src_expansion,
             m2l_result, src_rscale, tgt_rscale, sac):
 
-        m2l_result = fft(m2l_result, inverse=True, sac=sac)
         m2l_result = m2l_result[:2*tgt_expansion.order+1]
         return super().postprocess_local_exprs(tgt_expansion,
             src_expansion, m2l_result, src_rscale, tgt_rscale, sac)
 
+
 # }}} FourierBesselM2LWithFFT
+
+# {{{ helper
+
+def translation_classes_dependent_data_loopy_knl(tgt_expansion, src_expansion,
```

Comment:
Add a docstring to describe what this is about.

Reply:
Fixed in 07c1c93ab2576c9d6ac9eddb69232e12bdd1d125